gatsby-node.js

```diff
@@ -56,7 +71,23 @@ exports.onPostBuild = async function(
     await Promise.all(chunkJobs);
 
     if (settings) {
-      const { taskID } = await indexToUse.setSettings(settings);
+      // Account for forwardToReplicas:
+      const extraModifiers = forwardToReplicas ? { forwardToReplicas } : {};
+
+      let adjustedSettings = {};
+
+      // If we're building replicas, we don't want to add them to temporary indices
+      if (currentIndexIsTempIndex && settings.hasOwnProperty('replicas')) {
```

Comment:
is there a reason why you only filter out the `replicas` property when it exists? We could filter it out in all cases

can replace `currentIndexIsTempIndex` with `indexToUse === tmpIndex` and leave the previous code as-is

Reply:
Thanks for the `tmpIndex` check suggestion!

In regards to destructuring `replicas` off `settings`, I was under the impression that I couldn't destructure something that didn't exist... and, if I tried to do so, it would return an error... It's highly possible that I'm wrong about that... but, I figured this was the safest way to ensure that we don't try to destructure something that doesn't exist on `settings`.

Reply:
you can destructure something which isn't set on an object, and then it will be undefined, so no worries, it's actually a pattern I use quite often to filter out unwanted keys

Reply:
Ah, okay. 👍I'll push an update for that, then.

Reply:
@Haroenv thanks so much for the comments. 🙂I just pushed up a fix for your last one. 🙂